### PR TITLE
Add fan controls to pco_sdk/pco_sdk_cam

### PR DESCRIPTION
**Pull request by zacsimile**
Added these to test if my camera's fan was broken (it was), and they could be generally useful.

### CODE REVIEW

File: PYME/Acquire/Hardware/pco/pco_sdk_cam.py [hunk 1]
```diff
@@ -360,6 +360,13 @@ def SetCCDTemp(self, temp):
         self._ccd_temp_set_point = temp
         self._get_temps()
 
+    def GetFanModeSpeed(self):
```
Comment (David-Baddeley) [inline PYME/Acquire/Hardware/pco/pco_sdk_cam.py]: There's already a `SetFan(self, state)` method on the IXon camera - we should probably try adopt a consistent signature between cameras. In this case `state` is either 0, 1, or 2 (effectively an enum, meaning off, low, high). The simplest would probably be to adopt `SetFan(self, state)` across all cameras where `state` had a camera-specific meaning (as long as it was easily serialisable as, e.g. json). A little more work would be to define consistent states across cameras. For the PCO cameras, does the speed take a fixed set of values (effectively an enum), or is it variable across a range)?

Comment (David-Baddeley) [reply PYME/Acquire/Hardware/pco/pco_sdk_cam.py]: or to make the low level camera-api `_set_fan_mode_speed()` a private instance of the PCO class.

Comment (David-Baddeley) [reply PYME/Acquire/Hardware/pco/pco_sdk_cam.py]: Cancel above question re what the PCO SDK takes - you've covered it in the docstring

Comment (David-Baddeley) [reply PYME/Acquire/Hardware/pco/pco_sdk_cam.py]: Short term solution is probably to just make the methods "private" - i.e. not part of the camera API with a comment to revisit if/when we want API control. As I understand it, the main rational for adding them at the moment is testing.

Comment (zacsimile) [reply PYME/Acquire/Hardware/pco/pco_sdk_cam.py]: Yeah, they're just for testing. I'll just convert them to private.

File: PYME/Acquire/Hardware/pco/pco_sdk_cam.py [hunk 2]
```diff
@@ -360,6 +360,13 @@ def SetCCDTemp(self, temp):
         self._ccd_temp_set_point = temp
         self._get_temps()
 
+    def _get_fan_mode_speed(self):
+        mode, speed = pco_sdk.get_fan_control_parameters(self._handle)
+        return mode, speed
+
+    def _set_fan_mode_speed(self, mode, speed):
```
Comment (David-Baddeley) [inline PYME/Acquire/Hardware/pco/pco_sdk_cam.py:367]:
```suggestion
    def _set_fan_mode_speed(self, mode, speed):
        # TODO - unified fan control interface for all cameras (see also AndorIxon `SetFan()` and comments on PR1135)
```

**Pull request merged**